x-pack/legacy/plugins/siem/server/types.ts

```diff
@@ -13,3 +14,11 @@ export interface LegacyServices {
   config: Legacy.Server['config'];
   route: Legacy.Server['route'];
 }
+
+declare module 'src/core/server' {
```

Comment:
@rudolf I'd appreciate your review on this one (and the underlying client). We're currently the only consumer of this, and I don't expose the client in our plugin contract, but I can (should?) do that in the final migration PR.

Reply:
Would plugins depending on SIEM ever need to use this client? If so, you should export the types so that consumers can also get these.

Reply:
https://github.com/elastic/kibana/pull/58292/commits/eb932978dd40ddf1b3f3a2443e997e7a59f1f3a2

Reply:
FYI: https://github.com/elastic/kibana/issues/59250 was added to track the remaining work for these typings.